packages/dev/scripts/README.md

```diff
@@ -0,0 +1,25 @@
+# @blockly/dev-scripts [![Built on Blockly](https://tinyurl.com/built-on-blockly)](https://github.com/google/blockly)
+
+This package includes scripts and configuration used by Blockly extension
```

Comment:
maybe just choose between Blockly extensions or Blockly packages?

Reply:
Good point.

Reply:
Actually I agree with the standardization, but I actually think we should call them: 
extension packages.

Reply:
Ok, any reason? I'm fine either way, just thinking that we have everything under packages and don't really seem to use extension anywhere else.

Reply:
Blockly extension, could be confused core Blockly extensions: https://developers.google.com/blockly/guides/create-custom-blocks/extensions

Maybe ``Blockly package`` ain't too bad. They are npm packages afterall.
Under packages, we're referring to reach item by its type (plugin, field, block).

Reply:
Problem with Blockly package is that it doesn't really tell me anything about it being an extension on top of Blockly.

Reply:
I'm open to ideas.